phpspec.yml.ci

```diff
@@ -0,0 +1,10 @@
+suites:
+  data_import_suite:
+    namespace: Ddeboer\DataImport
+    psr4_prefix: Ddeboer\DataImport
+extensions:
+  - PhpSpec\Extension\CodeCoverageExtension
+formatter.name: pretty
+code_coverage:
```

Comment:
I guess some classes must be tested with phpunit if we want code coverage for them as well. However I usually think that using the appropriate tools is more important than code coverage. In this case, most readers/writers can only be integration tested by behat. The rest of the library can be fully (spec) tested by phpspec, so I think those tests could be removed from phpunit.